proc-macros/src/helpers.rs

```diff
@@ -165,6 +165,20 @@ pub(crate) fn is_option(ty: &syn::Type) -> bool {
 	false
 }
 
+/// Iterates over all Attribute's and parses only the attributes that are doc comments.
+///
+/// Note that `doc comments` are expanded into `#[doc = "some comment"]`
+/// Thus, if the attribute starts with `doc` => it's regarded as a doc comment.
+pub(crate) fn extract_doc_comments(attrs: &[syn::Attribute]) -> TokenStream2 {
+	let docs = attrs.iter().filter_map(|attr| match attr.parse_meta() {
```

Comment:
I'd flip the order here: first check if `attr.path` is a `"doc"` ident, then do the more expensive parse meta to see if it's a name value pair with literal string value.

Reply:
ty, will fix